src/components/DataCard/DataCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { type } from "os";
+import React from "react";
+import {
+  DataCardContainer,
+  DataCardHeader,
+  BottomLineStyle,
+} from "./DataCard-style";
+import chartImg from "../../assets/Icons/chart.svg";
+import { colors } from "../../Utils/Constances";
+
+type IDataCard = {
+  title: string;
+};
+
+const DataCard = (props: IDataCard) => {
+  const { title } = props;
+
+  return (
+    <DataCardContainer>
+      <DataCardHeader>
+        {title}
+        <BottomLineStyle />
+      </DataCardHeader>
+      <div
```

Comment:
I think it will be better to get out that part to a separate component, and then you could use that for any case of 'no data'. What do you think?